src/com/ms/silverking/cloud/dht/daemon/MessageModule.java

```diff
@@ -585,12 +586,12 @@ public PrimarySecondaryIPListPair getReplicaListPair(long context, DHTKey key, R
             PrimarySecondaryIPListPair  replicaListPair;
             
             if (debug) {
-                System.out.println("\t\tgetReplicaListPair "+ key);
+                Log.fineAsyncf("getReplicaListPair: %s ", key);
```

Comment:
debug is off by default. Make Log.warningAsyncf so that the debug flag has an effect